catalog/resource_schema_test.go

```diff
@@ -40,6 +51,7 @@ func TestCreateSchema(t *testing.T) {
 		name = "a"
 		catalog_name = "b"
 		comment = "c"
+		owner = "administrators"
```

Comment:
same comment about not changing tests and introducing new ones

Reply:
new tests